values-schema.yaml

```diff
@@ -218,6 +218,40 @@ definitions:
         type: string
     title: Container image
     type: object
+  jobs:
+    description: |
+      You may pass an array of Job objects. The Kubernetes API accepts a Job specification, 
+      which is meant for run-to-completion apps (e.g. batch Jobs).
+    items:
+      description: |
+        You may pass a Job object. You must pass the following properties:
+
+          enabled: (type boolean)
+          env: (type array of strings, pattern-matched)
+          image: (type string, pattern-matched)
+          init: (type object)
+            image: (type string, pattern-matched)
+            script: (type string)
+      properties:
```

Comment:
There is also job type property `type`.
If `type=cronjob` then the `schedule` property is used.
You can find this at `charts/jobs/templates/cronjob.yaml`

Reply:
Ok, good to know!

Reply:
I see that I can add a few more like spec.mounts, should I?

Reply:
I think there already exists that option to use mounts in the job chart. (I believe I added it.) Please take a look

Reply:
@Morriz I mean to add it to the `values-schema.yaml`